kienstra/gutenberg
# Convert the `ColorPalette` component to TypeScript

## What?
Convert the `ColorPalette` component to TypeScript

## Why?
As part of an effort to convert `@wordpress/components` to TypeScript

## How?
By adding types to the `ColorPalette` component and its story

## Testing Instructions
1. `npm run storybook:dev`
2. Verify that the `ColorPalette` component works the same as before

## Screenshots or screencast 
![color-palette](https://user-images.githubusercontent.com/4063887/193477342-2ef0e1fe-9e06-4615-8dfb-7a95515fff4e.gif)
